index.bs

```diff
@@ -9888,14 +9949,6 @@ Constructors</h5>
 			options: Optional initial parameters value for this {{AudioWorkletNode}}.
 		</pre>
 
-		Once the construction of the node is completed,
-		<dfn dfn>processor construction data</dfn> will be created and
-		transferred to the matching {{AudioWorkletProcessor}}'s
-		{{AudioWorkletProcessor()|constructor}}. This data contains
-		a set of objects and references that is required for
-		{{AudioWorkletProcessor}}'s construction. See the step 12 in
-		the algorithm below.
-
 		When the constructor was called, the user agent MUST perform the
```

Comment:
Nit: "was called" -> "is called".  But since this isn't part of this PR, you can ignore this.